front/src/app/components/headerComponent/HeaderComponent.tsx

```diff
@@ -0,0 +1,34 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTelegram } from "@fortawesome/free-brands-svg-icons";
+import Status from "../statusComponent/StatusComponent";
+import { StatusProps } from "../statusComponent/type";
+
+export default function HeaderComponent({
+  error,
+  data,
+  connectionRestored,
+  setConnectionRestored,
+  lastMessage,
+}: StatusProps) {
+  return (
+    <div className="flex justify-evenly w-full">
```

Comment:
I would suggest different spacing for the header with ...

```suggestion
    <div className="flex justify-evenly w-full p-2">
```